examples/hikari_lightbulb_basic_queue/music_plugin.py

```diff
@@ -0,0 +1,257 @@
+        # it will search the query on youtube.
+        query_information = await self.bot.data.lavalink.auto_search_tracks(query)
+
+        if not query_information.tracks:  # tracks is empty
+            await ctx.respond(
+                "Could not find any video of the search query."
+            )
+            return
+
+        try:
+            # `.requester()` To set who requested the track, so you can show it on now-playing or queue.
+            # `.queue()` To add the track to the queue rather than starting to play the track now.
+            await self.bot.data.lavalink.play(
+                ctx.guild_id, query_information.tracks[0]
+            ).requester(ctx.author.id).queue()
+        except lavasnek_rs.NoSessionPresent:
+            await ctx.respond(f"Use `{PREFIX}join` first")
+            return
+
+        await ctx.respond(
+            f"Added to queue: {query_information.tracks[0].info.title}"
+        )
+
+
+    @lightbulb.check(lightbulb.guild_only)
+    @lightbulb.command()
+    async def stop(self, ctx):
+        """Stops the current song (skip to continue)."""
+
+        await self.bot.data.lavalink.stop(ctx.guild_id)
+        await ctx.respond("Stopped playing")
+
+
+    @lightbulb.check(lightbulb.guild_only)
+    @lightbulb.command()
+    async def skip(self, ctx):
+        """Skips the current song."""
+
+        skip = await self.bot.data.lavalink.skip(ctx.guild_id)
+        node = await self.bot.data.lavalink.get_guild_node(ctx.guild_id)
+
+        if not skip:
+            await ctx.respond("Nothing to skip")
+        else:
+            # If the queue is empty, the next track won't start playing (because there isn't any),
+            # so we stop the player.
+            if not node.queue and not node.now_playing:
+                await self.bot.data.lavalink.stop(ctx.guild_id)
+
+            await ctx.respond(f"Skipped: {skip.track.info.title}")
+
+
+    @lightbulb.check(lightbulb.guild_only)
+    @lightbulb.command()
+    async def pause(self, ctx):
+        """Pauses the current song."""
+
+        await self.bot.data.lavalink.pause(ctx.guild_id)
+        await ctx.respond("Paused player")
+
+
+    @lightbulb.check(lightbulb.guild_only)
+    @lightbulb.command()
+    async def resume(self, ctx):
+        """Resumes playing the current song."""
+
+        await self.bot.data.lavalink.resume(ctx.guild_id)
+        await ctx.respond("Resumed player")
+
+
+    @lightbulb.check(lightbulb.guild_only)
+    @lightbulb.command(name='now-playing', aliases=['np'])
+    async def now_playing(self, ctx):
+        """Gets the song that's currently playing."""
+        node = await self.bot.data.lavalink.get_guild_node(ctx.guild_id)
+        # You could create a queue command buy iterating over `node.queue`.
+        if node.queue:
+            current = node.queue[0].track
+            await ctx.respond(f"Now Playing: {current.info.title}")
+        else:
+            await ctx.respond(f"The queue is empty")
```

Comment:
Be consistent with pure hikari (rebase) and hata.

```suggestion
    node = await self.bot.data.lavalink.get_guild_node(ctx.guild_id)

    if not node or not node.now_playing:
        await ctx.respond("Nothing is playing at the moment.")
        return

    # for queue, iterate over `node.queue`, where index 0 is now_playing.
    await ctx.respond(f"Now Playing: {node.now_playing.track.info.title}")
```

Reply:
Resolved